### PR TITLE
Expose volume inodes to Stackdriver sink

### PR DESCRIPTION
**Pull request by wangzhen127**
This PR exposes volume inodes to Stackdriver sink

### CODE REVIEW

File: metrics/sinks/stackdriver/metadata.go [hunk 1]
```diff
@@ -279,6 +279,18 @@ var (
...
+	legacyVolumeInodesMD = &metricMetadata{
+		MetricKind: google_api5.MetricDescriptor_GAUGE,
+		ValueType:  google_api5.MetricDescriptor_INT64,
+		Name:       "container.googleapis.com/internal/container/volume/inodes",
+	}
+
+	legacyVolumeInodesFreeMD = &metricMetadata{
+		MetricKind: google_api5.MetricDescriptor_GAUGE,
+		ValueType:  google_api5.MetricDescriptor_INT64,
+		Name:       "container.googleapis.com/internal/container/volume/inodes_free",
```
Comment (dashpole) [inline metrics/sinks/stackdriver/metadata.go]: is this only for volumes, or does this cover container inode metrics as well?

Comment (wangzhen127) [reply metrics/sinks/stackdriver/metadata.go]: Updated to include node level inode metrics now. PTAL

**Comment by dashpole** [comment]
lgtm

**Review by bmoyles0117 — APPROVED** [review]
I apologize for the feedback latency, thank you for the contribution!

**Pull request merged**